integration-testing/src/fixtures/common.py

```diff
@@ -98,24 +96,27 @@ def validators_data(config):
 
     logging.info(f"Using validator keys: {validator_keys}")
 
-    with create_bonds_file(validator_keys) as bonds_file:
-        yield (bonds_file, validator_keys[0], validator_keys[1:])
-
-
-
-@pytest.fixture(scope="package")
-def bootstrap(docker, docker_network, config, validators_data):
-    bonds_file, bootstrap_keys, _ = validators_data
-    node = create_bootstrap_node(docker, docker_network, bonds_file, bootstrap_keys, config.rnode_timeout)
-
-    yield node
-
-    node.cleanup()
-
-@pytest.fixture(scope="package")
-def started_bootstrap(config, bootstrap):
-    wait_for( string_contains( node_logs(bootstrap),
-                               "coop.rchain.node.NodeRuntime - Listening for traffic on rnode"),
-              config.node_startup_timeout,
-              "Bootstrap node didn't start correctly")
-    yield bootstrap
\ No newline at end of file
+    with bonds_file(validator_keys) as f:
+        yield (f, validator_keys[0], validator_keys[1:])
```

Comment:
Can this be made into a namedtuple/dataclass to have clarity what the tuple elements are?

Reply:
good point